src/Campaigns/Blocks/CampaignGoal/block.json

```diff
@@ -19,5 +19,7 @@
     },
     "textdomain": "give",
     "editorScript": "file:../../../../build/campaignGoalBlock.js",
```

Comment:
Isn't this a leftover? Once all blocks are being compiled in a single script file, I believe this isn't necessary. Also, there's no mention this file being compiled anywhere.

Reply:
Resolved